modules/alloy/src/demo/ts/ephox/alloy/demo/InlinesDemo.ts

```diff
@@ -117,7 +117,7 @@ export default (): void => {
       Positioning.position(sink, {
         anchor: 'submenu',
         item
-      }, submenu);
+      }, submenu, Optional.none());
```

Comment:
It's a bit messy that it's another argument, but there's not really a strong case to make it part of the other arguments either. Positioning things just requires too much flexibility, I guess. Some things you know ahead of time, and others you don't.

Reply:
I briefly considered trying to make some sort of options-thing, but with only one setting it seemed a bit much. Maybe if further options are added in the future?